lib/cmd/fh3/admin/domains/check.js

```diff
@@ -23,8 +23,10 @@ module.exports = {
     return "/box/api/domains/check?domain=" + domain;
   },
   'method' : 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createObjectTable(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
+      params._table = common.createObjectTable(params);
+    }
     return cb(null, params);
```

Comment:
This solution seems to be ok, but how about extending callback function `cb` to accept table. 
Having `_table` as part of result seems like odd hack rather than solid implementation.

I think we can do something like:

```
 return cb(null, params, customTable);
```

For generic table we should not really have this as boilerplace because actual handler for this cb has fully access to command line arguments = argv.json can be used as flag in code that format output and we do not need to set _table to have it done using generic table handler.
I hope that's clear. 

Note that this is only suggestion. Current implementation works but has one particular drawback:
Every command need to explicitly handle table creation which may be really hard to enforce at later stage.